pkg/config/config_test.go

```diff
@@ -23,9 +24,12 @@ func setupConf() Config {
 
 func setupConfFromYAML(yamlConfig string) Config {
 	conf := NewConfig("datadog", "DD", strings.NewReplacer(".", "_"))
-	conf.SetConfigType("yaml")
-	conf.ReadConfig(bytes.NewBuffer([]byte(yamlConfig)))
 	initConfig(conf)
+	conf.SetConfigType("yaml")
+	e := conf.ReadConfig(bytes.NewBuffer([]byte(yamlConfig)))
+	if e != nil {
+		fmt.Println(e)
```

Comment:
Should this be `log.Fatal` ?

Reply:
`log.Fatal` would abort the tests, I've changed it to `log.Println`

Reply:
Oh, sorry. I thought we were meant to fail here.